README.md

```diff
@@ -2,14 +2,15 @@
 
 # PeopleDepot
 
-PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot provides backend infrastructure to provide a single source of truth on people, program areas, and project data for all Hack for LA projects. PeopleDepot uses a Django database and Django REST Framework that acts as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA, Civic Tech Index, Tables).
+PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot aims to provide a single source of truth as the backend infrastructure and database store for all Hack for LA projects, including data about people, program areas, and projects. PeopleDepot uses Django for its database and REST Framework and serves as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA Website, Civic Tech Index, Tables).
 
 ## Project context
 
-The hardest part of running an all volunteer open source project organization is managing the flow of information to all the people and projects that need it.  It can end up taking more time than the projects themselves. This project seeks to create an infrastructure that will automate everything that can be automated and do away with manual storage of duplicate information across projects.
+The hardest part about running a large organization using only free or open source tools and technologies is how to manage the flow of information and provide relevant info to all the people and projects that need it. Managing multiple databases inefficiently can end up taking more time than the projects themselves. This project seeks to create a maintainable database infrastructure that is synchronized. 
 
+In the process, it should allow for further automation and do away with manual storage of duplicate information across projects, which includes:
 - Recruiting members (Website: Project info and meeting times)
-- Onboarding members to resources (Tables:GitHub, Google Calendar, Google Drives, etc.)
+- Onboarding members to resources (Tables: GitHub, Google Calendar, Google Drives, etc.)
```

Comment:
tables and github are separate items

google tables

Reply:
I thought Tables was a project. Should we say Google Sheets? (Google Tables would sound funny.)

Reply:
> I thought Tables was a project. Should we say Google Sheets? (Google Tables would sound funny.)

Never mind, Google Tables is a thing 👀